powershell/VstsTaskSdk/ToolFunctions.ps1

```diff
@@ -61,7 +61,9 @@ function Invoke-TaskScript {
         [scriptblock]$ScriptBlock)
 
     try {
+        $global:ErrorActionPreference = 'Stop'
```

Comment:
Previous the error action preference was set within Merge-Pipelines. I just moved it out for readability. Also it is now set at the global level. This is related to a problem I ran into while testing the Azure PowerShell task changes. The module-scope inherits from the global scope, so errors from within module-scopes weren't getting treated as terminating errors.

I'm still trying to figure out why I didn't catch this earlier. I remember specifically testing this at one point. I need to add a unit test.